prow/cluster/monitoring/mixins/prometheus/hook_alert.libsonnet

```
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'abnormal webhook behaviors',
+        rules: [
+          {
+            alert: 'no-webhook-calls',
+            // utc time of 9a-5p PDT: with some offset on day_off_week
```

Does the offset basically mean monday-friday? If so, how about:
```suggestion
            // Monday-Friday 9am-5pm PDT (in UTC)
```

---

Done.

you are correct.
The reason for PDT 5-9 is that we feel that most of cluster-admins/googlers are from that timezone.
However, i cannot find a way to tell prometheus to use a specific tz.
So we wont have alerts on PDT Friday afternoon (which is already Saturday UTC).

---

Now (814d3a2). It is supposed to be precisely Monday-Friday 9am-5pm PDT (in UTC).